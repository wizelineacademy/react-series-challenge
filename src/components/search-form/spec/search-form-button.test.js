import React from 'react';
import renderer from 'react-test-renderer';

import { SearchButton } from '../';

describe('<SearchButton />', () => {
  test('renders data correctly', () => {
    const tree = renderer.create(<SearchButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
