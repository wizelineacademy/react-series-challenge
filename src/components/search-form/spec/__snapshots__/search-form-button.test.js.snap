// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchButton /> renders data correctly 1`] = `
<button
  className="sc-bdVaJa iJovzf"
/>
`;
